### DISCUSSION TITLE
Waiting list: Make the reservation not bound to a specific product but a category

### DISCUSSION THREAD

**Discussion by Unicorn-rzl** [pretix/pretix#1971] [Ideas]
The current implementation of the waiting list is product specific. Therefore, the users are not able to change to a different ticket category after they've been selected for a ticket from the waiting list. There are situations in which this makes perfectly sense. There are however other types of events where the selection what kind of ticket one buys is not as important as having a ticket at all. In this kind of scenario, the user is more likely to choose a cheaper ticket since choosing a higher price ticket has a diminishing marginal utility in comparison to the high marginal value of the cheapest ticket. It would be interesting to back this theory by numbers from multiple events (ticket distribution without vs. with waiting list).

Suggestion would be to allow the reservation not of specific tickets but of categories of tickets. ("This voucher allows to buy any ticket" instead of "This voucher allows to buy the pre-chosen ticket")

**Comment by raphaelm** [comment]
Yep, would be good to have this, just don't know yet how to approach this.

I'm not sure if categories are the right level to solve this on, as they are used very differently. "Any ticket" also doesn't work that well probably, as tickets can be *really* different. Best would probably to let the user choose "any of the list following products [checkboxes]".

Technically, it's also hard (but probably not impossible?) to build a waiting list across multiple quotas, so the natural selection would be by quota, but that is intransparent and unclear to the user.

**Comment by vincentization** [comment]
I know its been a while, but I still want to add a slight variation of this idea:
For event series it would be extremely helpful to have a general waiting list thats not bound to a certain date. This would not only help in judging the general demand for a certain event, but also to notify all people on the waiting list that a new date has been added
